src/protobuf_impl.rs

```diff
@@ -23,21 +23,8 @@ fn get_protoc() -> String {
     if let Ok(s) = check_protoc_version("protoc") {
         return s;
     }
-
     // The bundled protoc should always match the version
-    let protoc_bin_name = match (env::consts::OS, env::consts::ARCH) {
-        ("linux", "x86") => "protoc-linux-x86_32",
-        ("linux", "x86_64") => "protoc-linux-x86_64",
-        ("linux", "aarch64") => "protoc-linux-aarch_64",
-        ("linux", "powerpc64") => "protoc-linux-ppcle_64",
-        ("macos", "x86_64") => "protoc-osx-x86_64",
```

Comment:
The initial motivation is to support macos + aarch64. We can bundle the new binary but why?